### PR TITLE
VDS-75: Fix sonar issues

### PR DESCRIPTION
**Pull request by pushnitsa**
Description: 

--

QA-test: VDS-1710

Demo-test: VDS-1711

Download artifact URL: docker.pkg.github.com/VirtoCommerce/vc-demo-storefront/demo-storefront:1.13.0-pr-67-vds-75-c9b9ef6d

### CODE REVIEW

File: VirtoCommerce.Storefront/wwwroot/swagger/print.css [hunk 1]
```diff
@@ -416,7 +416,7 @@
             font-size: .85em;
             line-height: 1.2em;
             overflow: auto;
-            max-height: 200px;
```
Comment (trueboroda) [inline VirtoCommerce.Storefront/wwwroot/swagger/print.css:419]: How it is related to the sonar issues?

File: VirtoCommerce.Storefront.Model/Common/CollectionExtensions.cs [hunk 2]
```diff
@@ -19,10 +17,12 @@ public static class CollectionExtensions
...
         /// <exception cref="System.ArgumentNullException">An <see cref="System.ArgumentNullException"/> is thrown if <paramref name="collection"/> or <paramref name="items"/> is <see langword="null"/>.</exception>
-		public static ICollection<T> AddRange<T>(this ICollection<T> collection, IEnumerable<T> items)
+        public static ICollection<T> AddRange<T>(this ICollection<T> collection, IEnumerable<T> items)
         {
-            if (collection == null) throw new System.ArgumentNullException("collection");
-            if (items == null) throw new System.ArgumentNullException("items");
+            if (collection == null)
+                throw new System.ArgumentNullException("collection");
+            if (items == null)
+                throw new System.ArgumentNullException("items");
```
Comment (asvishnyakov) [inline VirtoCommerce.Storefront.Model/Common/CollectionExtensions.cs]: I think due to our guidelines we should use curly braces here

File: VirtoCommerce.Storefront/Controllers/BulkOrderController.cs [hunk 3]
```diff
@@ -130,16 +130,14 @@ private async Task<string[]> TryAddItemsToCartAsync(BulkOrderItem[] bulkOrderIte
             var addedSkus = new List<string>();
             foreach (var bulkOrderItem in bulkOrderItems)
             {
-                if (foundSkusProductsWithVariationsMap.TryGetValue(bulkOrderItem.Sku, out var product))
-                {
-                    if (await _cartBuilder.AddItemAsync(new Model.Cart.AddCartItem
+                if (foundSkusProductsWithVariationsMap.TryGetValue(bulkOrderItem.Sku, out var product) &&
+                    await _cartBuilder.AddItemAsync(new Model.Cart.AddCartItem
```
Comment (asvishnyakov) [inline VirtoCommerce.Storefront/Controllers/BulkOrderController.cs]: I think it's better to move this call into the separate variable

File: VirtoCommerce.Storefront/Infrastructure/HmacUtility.cs [hunk 4]
```diff
@@ -141,27 +141,23 @@ public static bool TryParse(string input, out ApiRequestSignature parsedValue)
             if (input != null)
             {
                 var parts = input.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
-                if (parts.Length == 3)
+                if (parts.Length == 3 && parts[2].Length == 64)
                 {
-                    if (parts[2].Length == 64)
+                    if (DateTime.TryParseExact(
```
Comment (asvishnyakov) [inline VirtoCommerce.Storefront/Infrastructure/HmacUtility.cs:146]: Why don't merge all 3?

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

**Review by asvishnyakov — COMMENTED** [review]
Also please mention cart validation testing in QA issue

**Review by trueboroda — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**